docs/notebooks/12Datastore_from_numpy_arrays.ipynb

```diff
@@ -27,7 +27,9 @@
     "import xarray as xr\n",
     "\n",
     "from dtscalibration import read_silixa_files\n",
-    "from dtscalibration import DtsAccessor   # noqa: E401\n",
+    "\n",
+    "# The following line introduces the .dts accessor for xarray datasets\n",
+    "import dtscalibration  # noqa: E401  # noqa: E401\n",
```

Comment:
Even in this notebook this is not needed.

You can remove this line and the notebook runs completely without errors. (same for all other ones)